core/base/inc/Rtypes.h

```diff
@@ -269,7 +269,7 @@ class ClassDefGenerateInitInstanceLocalInjector:
 #define _ClassDefBase_(name, id, virtual_keyword, overrd)                                                       \
 private:          \
    static_assert(std::is_integral<decltype(id)>::value, "ClassDef(Inline) macro: the specified class version number is not an integer.");                                                        \
-   virtual_keyword Bool_t CheckTObjectHashConsistency() const overrd                                            \
+   /** \cond HIDDEN_SYMBOLS */ virtual_keyword Bool_t CheckTObjectHashConsistency() const overrd                \
```

Comment:
Can the `\cond HIDDEN_SYMBOLS` span all these functions with a final `\endcond` in the end, instead of repeating it for each?

Reply:
@pcanal - what's your take, should we expose the `ClassDef` functions through doxygen or not? Or only some, like `Class_Name()`, `IsA()`, `Class()`?

Reply:
Most of the public functions in `ClassDef` should be published (in particular `IsA`, `Class`, `DeclFileName` are currently missing).

`Dictionary`, `StreamerNVirtual` and `ShowMembers` are debatable, they are part of the public interface but somewhat implementation detail but not without use.

`DeclFileLine` is debatable (only as in so far as the information is not that useful)

`ImplFileLine` and `ImplFileName` are deprecated and should be hidden.